Building sparseqr for pyodide 0.23.1

Hi,

I was looking into building sparseqr for pyodide 0.23.1, which has been broken since https://github.com/pyodide/pyodide/pull/3252 

**From source:**

I did not manage to build pyodide from source in my machine (Ubuntu20.04LTS un wsl2), either with or without docker. Both act very differently from what I was used to, asking whether I want to reverse patches:

```
make -C /home/gfo/pyodide/cpython
make[1]: Entering directory '/home/gfo/pyodide/cpython'
[ -d /home/gfo/pyodide/cpython/build/Python-3.11.2 ] || (mkdir -p /home/gfo/pyodide/cpython/build/; tar -C /home/gfo/pyodide/cpython/build/ -xf /home/gfo/pyodide/cpython/downloads/Python-3.11.2.tgz)
cat patches/*.patch | (cd /home/gfo/pyodide/cpython/build/Python-3.11.2 ; patch -p1)
patching file Modules/main.c
Reversed (or previously applied) patch detected!  Assume -R? [n]    
```
and then failing like this:
```
patch: **** Can't create file Modules/main.c.orig : Too many open files
patch: **** Can't create file Modules/main.c.orig : Too many open files
patch: **** Can't create file Modules/main.c.orig/bin/bash: line 1:  1745 Segmentation fault
patch -p1
make[1]: *** [Makefile:59: /home/gfo/pyodide/cpython/build/Python-3.11.2/.patched] Error 139
make[1]: Leaving directory '/home/gfo/pyodide/cpython' 
make: *** [Makefile:245: /home/gfo/pyodide/cpython/installs/python-3.11.2/lib/python3.11] Error 2   
```

**Out of tree:**

Using the out of tree build system `pyodide build`, I was not able to make it work either. The build system successfully generates a `sparseqr-1.2.1-py3-none-any.whl`, but this is not the end, as there is a compile step taking place at the first import that then fails. TBH, I do not know how @ryanking13 got around this issue in previous versions...

Sorry I could not go any further on my own...

--- a/packages/sparseqr/meta.yaml
+++ b/packages/sparseqr/meta.yaml
@@ -1,7 +1,6 @@
 package:
   name: sparseqr
   version: "1.2"
-  _disabled: true
   top-level:
     - sparseqr
 

